proto/pocket/proof/event.proto

```diff
@@ -47,9 +45,12 @@ message EventProofUpdated {
 // Event emitted after a proof has been checked for validity in the proof module's
 // EndBlocker.
 message EventProofValidityChecked {
-  pocket.proof.Proof proof = 1 [(gogoproto.jsontag) = "proof"];
+  // Next index: 6
+  reserved 1;
+  reserved 3;
+
+  pocket.proof.Claim claim = 5 [(gogoproto.jsontag) = "claim"];
```

Comment:
Can you add a TODO (probably nex to where we define the claim) to :

1. Dehydrate the SessionHeader
2. Dehydrate the root hash

Can be a consideration or just an outright TODO.

For now, just need to get the proofs out.